GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h

```diff
@@ -16,7 +16,34 @@
 
 #import <Foundation/Foundation.h>
 
-/// Helper object used to hold GIDAccountDetailType representing a list of
+NS_ASSUME_NONNULL_BEGIN
+
+/// An enumeration defining the types of account details Google can verify.
+typedef NS_ENUM(NSInteger, GIDAccountDetailType) {
+  /// User account detail for age over 18.
+  GIDAccountDetailTypeAgeOver18,
+};
+
+extern NSString *const kAccountDetailTypeAgeOver18Scope;
```

Comment:
Everything in a header file should have a doc comment: `/// String scope representing...`.